manager/src/main/resources/application.yml

```diff
@@ -23,6 +23,7 @@ spring:
     static-path-pattern: /**
   jackson:
     default-property-inclusion: ALWAYS
+    time-zone: ${TZ:Asia/Shanghai}
```

Comment:
due the https://github.com/apache/hertzbeat/pull/2122 support config time-zone in webui, can this be removed? @Calvin979 @cdphantom

Reply:
> due the #2122 support config time-zone in webui, can this be removed? @Calvin979 @cdphantom

Yes, it can be removed. @cdphantom